code/app/global_variables.py

```diff
@@ -41,31 +134,155 @@
 
 
 for idx in np.arange(len(LGL_list)):
-	LGL_list[idx] = af.arith.cast(af.Array(LGL_list[idx]), af.Dtype.f64)
-	pass
+	LGL_list[idx] = np.array(LGL_list[idx], dtype = np.float64)
+	LGL_list[idx] = af.interop.np_to_af_array(LGL_list[idx])
+
```

Comment:
We are not using Gaussian nodes for now. Please remove the variables/functions related
to Gaussian Quadrature. Keep it only if we are going to use it somewhere.